### PR TITLE
Fixed error that triggered error dialog in Display Daily dialog

### LINKED ISSUE #7016: Developer error under Climatic > Check data > Display daily dialogue
**Describe the bug**
This message appears when opening the dialog after the first run.

**To Reproduce**

1. Go to Climatic > Check data > Display daily dialogue
2. Fill any variable in the Elements multiple receiver and then Click Ok
3. Then re-open again the dialogue, you will get the message in the screenshot below.

image

### PR DESCRIPTION
**Pull request by EstherNjeriLiberatta**
Fixes  #7016 

@N-thony kindly review this. I think the error is now fixed.

### CODE REVIEW

File: instat/dlgDisplayDailyData.vb [hunk 1]
```diff
@@ -240,7 +240,7 @@ Public Class dlgDisplayDailyData
         ucrReceiverElement.SetLinkedDisplayControl(lblElement)
 
 
-        ucrChkMissing.AddToLinkedControls({ucrInputComboMissing}, {True}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
+        ucrChkMissing.AddToLinkedControls({ucrInputComboMissing}, {True}, bNewLinkedUpdateFunction:=True, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
```
Comment (N-thony) [inline instat/dlgDisplayDailyData.vb:243]: Do you need bNewLinkedUpdateFunction:=True?

File: instat/dlgDisplayDailyData.vb [hunk 2]
```diff
@@ -177,7 +177,7 @@ Public Class dlgDisplayDailyData
 
         lstCheckboxes.AddRange({ucrChkSum, ucrChkMax, ucrChkMin, ucrChkMean, ucrChkMedian, ucrChkIQR, ucrChkSumMissing})
 
-        ucrChkMissing.SetParameter(ucrInputComboMissing.GetParameter)
+        ' ucrChkMissing.SetParameter(ucrInputComboMissing.GetParameter)
```
Comment (N-thony) [inline instat/dlgDisplayDailyData.vb:180]: Remove? Same with other commented line

**Review by N-thony — CHANGES_REQUESTED** [review]
Under Table option. Can you fix this small design issue with missing checkbox going over its groupbox
image

**Review by rdstern — DISMISSED** [review]
seems ok now

File: instat/dlgDisplayDailyData.vb (hunk 1 above)
Comment (EstherNjeriLiberatta) [reply instat/dlgDisplayDailyData.vb:243]: Hello @N-thony I've commented out the SetRCode for ucrInputComboMissing and replaced it with that. It does the same thing but it helps remove the confusion between the `ucrChkMissing` and `ucrInputComboMissing` controls.

**Comment by EstherNjeriLiberatta** [comment]
@N-thony I fixed the checkbox overlap. I also noticed an inconsistency in spelling of colour on the `Rug color` label. I fixed that too. Thanks.

@rdstern This is ready for review.

**Comment by lloyddewit** [comment]
> @N-thony I fixed the changes you requested. 

@EstherNjeriLiberatta I don't think you resolved @N-thony 's comments in `dlgDisplayDailyData.vb`, please could you double-check? thanks

**Comment by EstherNjeriLiberatta** [comment]
@lloyddewit No problem, thank you.

**Comment by EstherNjeriLiberatta** [comment]
Hello @N-thony please review the changes I've made.
@rdstern @africanmathsinitiative/developers This is ready for review.

**Review by rdstern — APPROVED** [review]
Great.

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**